frontend/src/lib/components/PayGateMini/PayGateMini.tsx

```diff
@@ -87,7 +98,7 @@ export function PayGateMini({
 
     const featureSummary = FEATURE_SUMMARIES[feature]
     const planRequired = FEATURE_MINIMUM_PLAN[feature]
-    let gateVariant: 'add-card' | 'contact-sales' | 'check-licensing' | null = null
+    let gateVariant: 'add-card' | 'contact-sales' | 'subscribe' | null = null
```

Comment:
this licensing stuff seems pretty outdated so I just switched it out with "subscribe" instead :o